hugo/content/exporter/_index.md

```diff
@@ -520,3 +479,151 @@ Collecting all per-query metrics into Prometheus could greatly increase storage
+
+#### Materialized View Metrics {#mat-view-metrics}
+
+With large databases/tables and some other conditions, certain metrics can cause excessive load. For those cases, materialized views and alternative metric queries have been made available. The materialized views are refreshed on their own schedule independent of the Prometheus data scrape, so any load that may be associated with gathering the underlying data is mitigated. A configuration table, seen below, contains options for how often these materialized views should be refreshed. And a single procedure can be called to refresh all materialized views relevant to monitoring.
+
+For every database that will be collecting materialized view metrics, you will have to run the {{< shell >}}setup_metric_views.sql{{< /shell >}} file against that database. This will likely need to be run as a superuser and must be run after running the base setup file mentioned above to create the necessary monitoring user first. 
+```
+psql -U postgres -d alphadb -f setup_metric_views.sql
+psql -U postgres -d betadb -f setup_metric_views.sql
+```
+The {{< shell >}}/etc/postgres_exporter/##/crontab.txt{{< /shell >}} file has an example entry for how to call the refresh procedure. You should modify this to run as often as you need depending on how recent you need your metric data to be. This procedure is safe to run on the primary or replicas and will safely exit if the database is in recovery mode.
+
+Configuration table {{< shell >}}monitor.metric_views{{< /shell >}}:
+
+|       Column       |       Description                                                |
+|--------------------|------------------------------------------------------------------|
+| view_schema     | Schema containing the materialized view |
+| view_name       | Name of the materialized view |
+| concurrent_refresh | Boolean that sets whether this materialized view can be refreshed concurrently (requires a unique index) |
+| run_interval       | How often this materialized view should have its data refreshed. Must be a value compatible with the PG interval type   |
+| last_run           | Timestamp of the last time this view was refreshed |
+| active             | Boolean that sets whether this view should be refreshed when the procedure is called |
+| scope              | Whether the data contained in the view is per-database or instance-wide. Currently unused |
+
+You are also free to use this materialized view system for your own custom metrics as well. Simply make a materialized view, add its name to the configuration table and ensure the user running the refresh has permissions to do so for your view(s).
+
+##### PGBouncer
+
+In order to monitor pgbouncer with postgres_exporter, the pgbouncer_fdw maintained by CrunchyData is required. Please see its repository for full installation instructions. A package for this is available for Crunchy customers.
```

Comment:
proper case pgbouncer in this line and line 593?

also shouldn't it be 'Crunchy Data customers' ?

Reply:
fixed